tracer/src/Datadog.Trace/Activity/Handlers/IgnoreActivityHandler.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="IgnoreActivityHandler.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using Datadog.Trace.Activity.DuckTypes;
+
+namespace Datadog.Trace.Activity.Handlers
+{
+    /// <summary>
+    /// Ignore Activity Handler catches existing integrations that also emits activities.
+    /// </summary>
+    internal class IgnoreActivityHandler : IActivityHandler
+    {
+        private static readonly string[] SourcesNames =
+        {
+            "Couchbase.DotnetSdk.RequestTracer",
+            "HttpHandlerDiagnosticListener",
+            "MassTransit",
```

Comment:
We don't have an existing integration for MassTransit, so we shouldn't ignore the source

Reply:
I have to check, because I created that list while I was developing and extracted from the logs I was receiving...

Reply:
I removed it here: https://github.com/DataDog/dd-trace-dotnet/pull/2446/commits/35b11371221bbb861428b0c9be1e916f2e9e69ce, let's check after CI

Reply:
worked!